### ISSUE TITLE
📌To do list

### ISSUE THREAD

**Issue by snowie2000** [snowie2000/mactype#535]
Here I am trying to list everything we are planning to do in the working version or versions in the future including new features and bug fixes.

Tasks completed will be marked with a strikethrough.

## Core tasks:

| Quest  | Status | Version | Issues related |
| ------------- | ------------- | -------- |  --------- |
| More global options | planned | -- | #475 |
| Custom installation location support | planned | -- | #530 |
| Exit mactray programatically | planned | -- | #521 |
| Deal with built-in fontsubstitution | testing| -- | #550 |
| Additional MacType.ini exclusions | in progress | -- | #523 |
| MacType incorrect language | in progress |  -- | #527 |
| Untranslated Chinese | in progress | -- | #502 |
| Super OTC support in MacTunner | planned | -- | #463   |
| UTF8 with BOM profile support | testing| -- | https://github.com/snowie2000/mactype/issues/564#issuecomment-507299893|
| MacTuner > INI reworking | planned | -- | #551 |
| DirectWrite in all Profiles | planned | -- | #588 |
| Exclude/Include a folder | testing| -- | #611 |
| WMI bug fixes | planned | -- | #653 |
|-------------------------|----------|--------|---------|
| ~~beta6 rendering changed~~ | completed  | beta6 | #550 |
| ~~`IncludeFont` feature~~  | completed | beta6 | #517 |
| ~~Chrome crash on Win8~~ | completed | beta6  | #522 |
| ~~MacTray crash report~~  | completed| beta6 | #532 |
| ~~Better asm loader~~ | completed | beta6  | #15 |
| ~~Installer deletes old 32-bit~~| completed | beta6  | #501 |

## Non-core tasks:

| Quest  | Status | Version | Issues related |
| ------------- | ------------- | -------- |  --------- |
| More informative website | help wanted | -- |  |
| MacType Tuner rebuild | help wanted | -- |  |
| MacType Wizard rebuild | help wanted | -- |  |
| Discord server | discussing | -- | #563  |

**Comment by sammilucia** [comment]
Thanks @snowie2000, good call. The new installer is in the git and should now clean up old 32-bit installations but will also need testing. I'll add other items I'm responsible for later...

**Comment by SHN3RT** [comment]
This project is dead?

**Comment by sammilucia** [comment]
@SHN3RT no... we just need more developers. the few people who work on MacType (and it's mostly @snowie2000) do it in our spare time.

MacType is very reliable and a lot of people use it (more than 10k). Was there something specific you were looking for?

**Comment by SHN3RT** [comment]
@sammilucia I'm not good at programming but I would translate Mactype from English to Polish. Is it an option to volunteer with translation?
